### PR TITLE
List

### LINKED ISSUE #19: List content
Many pages in AEM use a Generic List component in two different ways:
Variant 1: A two-column layout with icon + links or icon + text, this is on the homepage and a text variant is on the careers page

image

image

Variant 2: Text-only (no icons) -- this is also on the Careers page
image

There is a variant called "2 col" for two-columns in desktop view but we might need to see if there is even such a thing as a 1 column variant before implementing that -- it might make more sense to generally assume 2-column is the default behavior.

- https://www.stewart.com/en.html
- https://www.stewart.com/en/about-stewart/careers.html

### PR DESCRIPTION
**Pull request by arcreddy**
Fix #19

Updated https://main--stewart--hlxsites.hlx.page/tools/sidekick/library/library.html

Test URLs:
- Before: https://main--stewart--hlxsites.hlx.page/drafts/import/en/about-stewart/careers
- After: https://list--stewart--hlxsites.hlx.page/drafts/import/en/about-stewart/careers

### CODE REVIEW

File: blocks/list/list.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  [...block.children].forEach((ele) => {
+    list.append(getListItemRestructured(ele.querySelector(':scope > div')));
+  });
+
+  block.replaceChildren(list);
+
+  const section = block.closest('.section');
+
+  if (section.classList.value.split(' ').includes('white-card')) {
```
Comment (shsteimer) [inline blocks/list/list.js:63]: curious why this is section metadata and not a block variant?

File: blocks/list/list.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  return listItem;
+}
+
+/**
+ * loads and decorates the list
+ * @param {Element} block The list block element
+ */
+export default async function decorate(block) {
+  const list = document.createElement('ul');
+  list.className = 'ul';
```
Comment (shsteimer) [inline blocks/list/list.js:53]: ??

Comment (arcreddy) [reply blocks/list/list.js:53]: Short of a better class name. Will think and give a reasonable one(probably better one)

File: blocks/list/list.js (hunk 1 above)
Comment (arcreddy) [reply blocks/list/list.js:63]: The white card is a variant of the whole section not just the list block. So, I added it to the Section.

File: blocks/list/list.js (hunk 2 above)
Comment (shsteimer) [reply blocks/list/list.js:53]: I'd probably just use `.list` not a deal breaker, just found it curious. If there is no good class name, an element selector is perfectly acceptable as well IMHO

File: blocks/list/list.js (hunk 1 above)
Comment (shsteimer) [reply blocks/list/list.js:63]: hmmm, ok. I think we might want to thing about this a bit broader.

Where else is the card style used aside from list? I assume columns, where each columns becomes a card, but anywhere else?

Also, can cards be other colors than white? do they inherit the background opacity? etc.

This is fine for now, but I suspect will need to be reviewed and modified later.

**Review by shsteimer — APPROVED** [review]

File: blocks/list/list.js (hunk 1 above)
Comment (arcreddy) [reply blocks/list/list.js:63]: For sure, will create a separate ticket to handle card

**Pull request merged**